forge-gui/res/cardsfolder/upcoming/their_number_is_legion.txt

```diff
@@ -0,0 +1,12 @@
+Name:Their Number Is Legion
+ManaCost:X B B B B
+Types:Sorcery
+A:SP$ Token | TokenAmount$ X | TokenScript$ b_2_2_a_necron_warrior | TokenTapped$ True | SubAbility$ DBGainLife | SpellDescription$ Create X tapped 2/2 black Necron Warrior artifact creature tokens, then you gain life equal to the number of artifacts you control. Exile CARDNAME.
+SVar:DBGainLife:DB$ GainLife | LifeAmount$ Y | StackDescription$ you gain life equal to the number of artifacts you control | SubAbility$ DBChange
+SVar:DBChange:DB$ ChangeZone | Origin$ Stack | Destination$ Exile | StackDescription$ Exile CARDNAME
```

Comment:
With the way you have this set up now, not sure you'll need this StackDescription at all. If you do need it, you'll need to just add `.` at the end